app-misc/ckb/ckb-0.4.2-r1.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-inherit desktop cmake-utils systemd
+inherit desktop cmake systemd
```

Comment:
Is desktop eclass used anywhere in this ebuild? It looks like a remnant from older versions